datamol/mol.py

```diff
@@ -10,6 +10,7 @@
 import random
 import itertools
 import hashlib
+import importlib_resources
```

Comment:
`importlib_resources` is a package that is only useful prior to py3.9 cuz for `>=3.9` you now have it in the stdlib with `importlib.resources`. It is still nice to fallback in case of python 3.8 being used (don't think everyone has switched to 3.9 yet).

You can use the below:

```python
try:
    import importlib.resources as importlib_resources
except:
    import importlib_resources
```